code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -103,6 +103,10 @@
 	for(var/obj/item/F in H)
 		if(F.light_range > 0)
 			extinguishItem(F)
+		if(ishuman(H))
+			var/mob/living/carbon/human/C = H
+			if(C.glowing)
```

Comment:
you can just do a check for the organ itself and force toggle it off instead, then you can kill off this var or move it into the organ itself.

No need to pollute all living mobs with a var tied to one organ.